pkg/bitwarden/cli.go

```diff
@@ -1,11 +1,14 @@
 package bitwarden
 
 import (
+	"bytes"
+	b64 "encoding/base64"
```

Comment:
Nit: I think I would prefer it if you didn't alias this because to me that looks like it was referring to a non-stdlib base64 later on in the code